jobserver/views/job_requests.py

```diff
@@ -206,23 +205,22 @@ def get(self, request, *args, **kwargs):
             )
         )
 
-        honeycomb_context_starttime = job_request.created_at - timedelta(days=2)
-
-        honeycomb_context_endtime = timezone.now()
-        if job_request.completed_at is not None:
-            honeycomb_context_endtime = job_request.completed_at + timedelta(days=2)
-
         context = {
             "honeycomb_can_view_links": honeycomb_can_view_links,
-            "honeycomb_context_starttime": honeycomb_context_starttime,
-            "honeycomb_context_endtime": honeycomb_context_endtime,
+            "honeycomb_links": {},
             "job_request": job_request,
             "project_definition": project_definition,
             "project_yaml_url": job_request.get_file_url("project.yaml"),
             "user_can_cancel_jobs": can_cancel_jobs,
             "user_can_cancel_jobs": can_cancel_jobs,
             "view": self,
         }
+
+        if honeycomb_can_view_links:
+            context["honeycomb_links"][
+                "Job Request concurrency"
```

Comment:
very open to better suggestions here

Reply:
Maybe just "job request"? We may add more visualizations in future